challenges-22-plus.yml

```
@@ -0,0 +1,192 @@
+lazy_frame_full_frame:
+  question: |
+    My animal rescue website has a page with very useful contact numbers that
+    people should use if they see a stray cat or dog. But, as usefull as this is,
+    I'd like to show this information at the bottom of every page!
+    
+    My footer looks like this:
+    
+    ```twig
+    <div id="footer">
+      <p>Emergency Contact Numbers:</p>
+
+      <turbo-frame id="contact-numbers" src="{{ path('contact_numbers') }}"></turbo-frame>
+    </div>
+    ```
+    
+    And my contact page has all of its content in the `body` block.
+    
+    ```twig
+    {% block body %}
+        <h1>Useful Contact Numbers</h1>
+        
+        <p>Here are some usefull contact numbers in case you are in an emergency and
+        in need of help!</p>
+        
+        <p><strong>Hope for Paws</strong>: +1 310-880-1416</p>
+        
+        <p><strong>National Mill Dog Rescue</strong>: +1 719-683-3334</p>
+        
+        <p><strong>Catskill Animal Sanctuary</strong>: +1 845-336-8447</p>
+    {% endblock %}
+    ```
+    
+    How can I make turbo only load *only* the contact information into my footer every
+    time I load a page?
+  explanation: |
+    One of the best aspects of lazy frames, is that you can load only parts of a page into
```

```suggestion
    One of the best aspects of frames, is that you can load only parts of a page into
```

I removed lazy... because this is a property of frames in general.